astroquery/skyview/tests/test_skyview_remote.py

```diff
@@ -15,11 +16,9 @@ def test_get_image_list():
 
 @remote_data
```

Comment:
The file is relatively small, about 360 KB, since the images get (down)sampled to 300 by 300 pixels by default (and could be reduced further server-side if wanted).